### ISSUE TITLE
Creating an overview-features.html which can be emailed

### ISSUE THREAD

**Issue by osuser2018** [damianszczepanik/cucumber-reporting#727]
Greetings!  I'm very grateful for the availability of this Cucumber Reporting.  Its truly the best one I've seen.  There is one issue, however, for which I'm hoping someone can provide some guidance and information.  

I have a compiled Jar file running through a Jenkins build.  However, the overview-features.html (and other html files) are not emailable because of the JavaScript and CSS dependencies... at least I have not been able to find a solution, scratching and clawing my way through online forums.  Ideally, it would be wonderful to email the great feature summary grid on overview-features.html.

Has there been any solutions developed to meet this need?  It seems emailing this great report would be a big need for folks wanting to broadcast the daily test results.  But as it is, one would need to jump through other hoops to view this information.

If there is any guidance on this issue which is available, I would greatly appreciate it.

**Comment by osuser2018** [comment]
Hi Damian,
Thanks for your response!

I am on version 3.16.0 currently (this version looks like it came out in April 2018).  I am not seeing the fix.  When I attach “overview-features.html” to a Jenkins email, the emailed report is unformatted.

Apparently I am missing something…  Can you explain what file I should attach to my Jenkins email?  Simply grabbing the “overview-features.html” and attaching to a Jenkins email does not seem to result in a formatted report.

Thanks!

**Comment by osuser2018** [comment]
That is correct.  When I attach the file (overview-features.html) through my Jenkins build setup to an email notification, it ultimately does not render the report  properly.  It is very easy to see that it lacks all formatting when I open the attachment from the sent email.  I assume that the HTML file itself lacks the styling code built into it and relies on external files for styling.

In Jenkins, I use a Post-Build action for the “Editable Email Notification”.  Within that setup, I add the file in the “Default Content” section.  Here is the line I use:   ${FILE,path="target/ overview-features.html "}

If there is anything else you would like to know about my setup, please let me know.

David

**Comment by FantaAI** [comment]
> I am on version 3.16.0 currently (this version looks like it came out in April 2018). I am not seeing the fix. When I attach “overview-features.html” to a Jenkins email, the emailed report is unformatted. Apparently I am missing something… Can you explain what file I should attach to my Jenkins email? Simply grabbing the “overview-features.html” and attaching to a Jenkins email does not seem to result in a formatted report.

Do you have an answer for the report? @osuser2018
